src/core.js

```diff
@@ -73,15 +77,15 @@ export function zoomRect(chart, p0, p1, transition = 'none') {
   const {options: {limits, zoom: zoomOptions}} = getState(chart);
```

Comment:
Same here. Might actually be better to pass the state to storeOriginalScaleLimits?